### PR TITLE
Translated Into locale language Gujrati (gu)

### PR DESCRIPTION
**Pull request by spielers**
Please have a look and merge it @eldadfux

### CODE REVIEW

File: app/config/locale/translations/gu.countries.php [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    'JP' => 'જાપાન',
+    'KZ' => 'કઝાકિસ્તાન',
+    'KE' => 'કેન્યા',
+    'KG' => 'કિર્ગીસ્તાન',
+    'KH' => 'કંબોડિયા',
+    'KI' => 'કિરીબતી',
+    'KN' => 'સેન્ટ કિટ્સ એન્ડ નેવિસ',
+    'KR' => 'દક્ષિણ કોરિયા',
+    'KW' => 'કુવૈત',
+
```
Comment (eldadfux) [inline app/config/locale/translations/gu.countries.php:102]: Please remove the extra space

File: app/config/locale/translations/gu.countries.php [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    'PA' => 'પનામા',
+    'PE' => 'પેરુ',
+    'PH' => 'ફિલિપાઇન્સ',
+    'PW' => 'પલાઉ',
+    'PG' => 'પપુઆ ન્યુ ગિની',
+    'PL' => 'પોલેન્ડ',
+    'KP' => 'ઉત્તર કોરિયા',
+    'PT' => 'પોર્ટુગલ',
+    'PY' => 'પેરાગ્વે',
+
```
Comment (eldadfux) [inline app/config/locale/translations/gu.countries.php:152]: Please remove the extra space

File: app/config/locale/translations/gu.countries.php (hunk 1 above)
Comment (spielers) [reply app/config/locale/translations/gu.countries.php:102]: @eldadfux That's come by default in Gujrati I have tried to remove the spaces but it changes the actual meaning of the word.

Comment (spielers) [reply app/config/locale/translations/gu.countries.php]: Done @eldadfux

File: app/config/locale/translations/gu.countries.php (hunk 2 above)
Comment (spielers) [reply app/config/locale/translations/gu.countries.php]: @eldadfux  Done

**Comment by eldadfux** [comment]
🥳

**Pull request merged**